test/boost/cql_query_test.cc

```diff
@@ -4313,3 +4315,198 @@ SEASTAR_TEST_CASE(test_rf_expand) {
         });
     });
 }
+
+/*!
+ * \brief this helper function changes all white space into a single space for
+ * a more robust testing.
+ */
+std::string normalize_white_space(const std::string& str) {
+  return std::regex_replace(" " + str + " ", std::regex("\\s+"), " ");
+}
+
+SEASTAR_TEST_CASE(test_describe_simple_schema) {
+    return do_with_cql_env_thread([] (cql_test_env& e) {
+        std::unordered_map<std::string, std::string> cql_create_tables {
+            {"cf", "CREATE TABLE ks.cf (\n"
+            "    pk blob,\n"
```

Comment:
nitpick: the indentation of this "cf" is different from the following ones.

Reply:
ok